README.md

```diff
@@ -5,10 +5,15 @@
 
 The Adobe Experience Platform Edge Network mobile extension allows you to send data to the Adobe  Edge Network from a mobile application. This extension allows you to implement Adobe Experience Cloud capabilities in a more robust way, serve multiple Adobe solutions though one network call, and simultaneously forward this information to the Adobe Experience Platform.
 
-The AEP Edge Network mobile extension is an extension for the [Adobe Experience Platform SDK](https://developer.adobe.com/client-sdks/documentation/) and requires the `AEPCore` and `AEPServices` extensions for event handling, as well as the `AEPEdgeIdentity` extension for handling identities, such as `ECID`.
+The AEP Edge Network mobile extension is an extension for the Adobe Experience Platform SDK and requires the `AEPCore` and `AEPServices` extensions for event handling, as well as the `AEPEdgeIdentity` extension for handling identities, such as `ECID`.
 
-To learn more about this extension, read the [Adobe Experience Platform Edge Network](https://developer.adobe.com/client-sdks/documentation/edge-network/) documentation.
+## Documentation
+
+To learn more about this extension, see the [Adobe Experience Platform Edge Network](https://developer.adobe.com/client-sdks/documentation/edge-network/) documentation.
+
+Additional information about SDK usage and architecture can be found in the [Documentation](Documentation) directory.
 
```

Comment:
Can we reword to something on these lines:

Information regarding AEP Edge Network extension implementation, API usage and architecture can be found in [Documentation](Documentation) directory

To Learn more about AEP Edge Network extension along with all the other AEP Mobile extensions on the official [AEP Mobile documentation](https://developer.adobe.com/client-sdks/documentation/edge-network/).